sk-ctrl/src/tests/controller_test.rs

```diff
@@ -24,6 +25,13 @@ fn opts() -> Options {
     }
 }
 
+enum WebhookState {
```

Comment:
Huh.  This is an interesting pattern I haven't seen before (defining an enum to classify different test states).  I.... think I like it?  Let's leave it in for now, I'm going to see if there are other places this is useful.